tests/test_couch/test_couch_parsers.py

```diff
@@ -14,7 +14,7 @@ def test_basic_log_parsing(self):
         metric_name, timestamp, request_time, attrs = parse_couch_logs(logging, SIMPLE)
 
         self.assertEqual(metric_name, 'couch.timings')
-        self.assertEqual(timestamp, 1446330723.0)
+        self.assertEqual(timestamp, 1446309123.0)
```

Comment:
i realized i think this keeps changing because it parses the timestamp in our local timezone haha. i've done once in boston and chicago too